doc/getting-started/apl-nuc.rst

```diff
@@ -84,6 +93,10 @@ complete this setup.
       | kernel-pk          | Run the Intel "PK" kernel(product kernel source)  |
       |                    | and enterprise-style kernel with backports        |
       +--------------------+---------------------------------------------------+
+      | desktop            | desktop application, Weston included              |
+      +--------------------+---------------------------------------------------+
+      | openssh-server     | include openssh                                   |
```

Comment:
a better description would be:

    server-side support for secure connectivity and
    remote login using the SSH protocol